lib/scimitar_schema_extension.rb

```diff
@@ -48,6 +48,49 @@ class OpenProjectNameComplexType < Scimitar::ComplexTypes::Base
 end
 
 module ScimitarSchemaExtension
+  AUTHENTICATION_SCHEMES = [
+    Scimitar::AuthenticationScheme.new(
+      type: "oauth2",
+      name: "OAuth2",
+      description: OpenProject::Static::Links.links[:sysadmin_docs][:scim_static_access_token_authentication_method][:href]
```

Comment:
:yellow_circle: One more chance for me to advertise that there is a helper for these kinds of calls to the static links:

```suggestion
      description: OpenProject::Static::Links.url_for(:sysadmin_docs, :scim_static_access_token_authentication_method)
```

The main advantage being that this just semantically says "I want that link", without revealing the underlying data structure. For example there was already experiments going on, where this method would then return different links based on the current user's locale.